SingleRepo.java

```diff
@@ -0,0 +1,15 @@
+public synchronized static SingletonRepo getInstance() {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;SingletonRepo&#34;, @ expected